writers/mixedbatchwriter/mixedbatchwriter.go

```diff
@@ -193,6 +193,8 @@ func (m *batchManager[A, T]) flush(ctx context.Context) error {
 		return err
 	}
 	m.batch = nil
+	clear(m.batch) // GC can work
```

Comment:
Aren't we setting `m.batch` to `nil` a bit before?

Reply:
typo, thanks!

Reply:
6ea725a8fafa3f38c1d07010395aa5e597e9c0fb